Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare v4.5.0

- [x] Latest Edge, Chrome, Firefox, Safari
  - [x] Chrome & Firefox on Android
- [x] [sync with Bootstrap](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Syncing-with-Bootstrap)'s release and probably wait for it (have a look at [Bootstrap #30709](https://github.com/twbs/bootstrap/pull/30709))
- [x] `npm run release-version 4.4.1 4.5.0` to bump version number
  - [x] `conventional-changelog -p angular -i CHANGELOG.md -s`
  - [x] and probably maintain a ship list: fixes #376
- [ ] merge (on `v4-dev` for now)
- [ ] tag your version
  - attach zip file
  - paste CHANGELOG in the release's description
  - copy `_gh_pages/` top level **files** (`index.html`, `redirects.json`, `sitemap.xml`…) and `_gh_pages/docs/4.5` to the `gh-pages` branch
  - ensure Algolia indexes new release content ([probably requires a PR](https://github.com/algolia/docsearch-configs/pull/811#issuecomment-481250552))
- [ ] publish on Nuget (@Lausselloic)
